package ch03;

public class f15_Exception {
	/*
	 * 	예외발생->프로그램 종료
	 * 	예외처리->프로그램 종료할지 계속할지 결정
	 * 
	 * 	1. try-catch-finally(생략가능)
	 * 		try{
	 * 			예외 발생 가능한 코드
	 * 		}
	 * 		catch(ArrayIndexOutOfBoundsException | InputMismathException e){
	 * 			예외 발생시 처리할 코드
	 * 			e.traceTrack();
	 * 		}
	 * 		finally{
	 * 			예외 상관없이 실행하는 코드
	 * 			메소드 예외처리할 때 주로 사용
	 * 			e.traceTrack();
	 * 		}
	 * 	2. throw
	 * 	3. throws
	 * 	4. try-with-resource
	 */
	public static void main(String[] args) {
		
	}

}
